test-cases/longevity/longevity-ndbench-12-nodes-multidc-4h.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 240
+
+prepare_write_cmd: >-
+  ndbench cli.clientName=CassJavaDriverGeneric ; numKeys=20971520 ; numValues=10 ; dataSize=512 ; cass.colsPerRow=10;
+  readEnabled=true; numReaders=100; numWriters=100 ; cass.writeConsistencyLevel=LOCAL_QUORUM ; generateChecksum=true;
+  cli.timeoutMillis=14400000; writeRateLimit=1350 ; readRateLimit=12000
+
+stress_cmd: >-
+  ndbench cli.clientName=CassJavaDriverGeneric ; numKeys=20971520 ; numValues=10 ; dataSize=512 ; cass.colsPerRow=10;
+  readEnabled=true; numReaders=100; numWriters=100 ; cass.writeConsistencyLevel=LOCAL_QUORUM ; generateChecksum=true;
+  cli.timeoutMillis=14400000; writeRateLimit=1350 ; readRateLimit=12000
+
+n_db_nodes: '6 6'
+n_loaders: '2 2'
+region_aware_loader: true
```

Comment:
remove, since it is currently relevant for YCSB

Reply:
Removed